src/diff/index.js

```diff
@@ -1,11 +1,39 @@
 import { EMPTY_OBJ, EMPTY_ARR } from '../constants';
 import { Component } from '../component';
 import { Fragment } from '../create-element';
-import { diffChildren } from './children';
+import { diffChildren, placeChild } from './children';
 import { diffProps, setProperty } from './props';
 import { assign, removeNode } from '../util';
 import options from '../options';
 
+function reorderChildren(newVNode, oldDom, parentDom) {
+	for (let tmp = 0; tmp < newVNode._children.length; tmp++) {
+		if (newVNode._children[tmp]) {
+			let vnode = newVNode._children[tmp];
+			vnode._parent = newVNode;
+			if (vnode._dom && vnode._dom.isConnected) {
```

Comment:
I think we can remove this `isConnected` now. I'm not sure why I added it in the first place lol 🤷‍♂️ Tests are passing locally for me without it.

```suggestion
			if (vnode._dom) {
```

Reply:
Glad we're not relying on this anymore - it isn't supported in IE11 anyway.